utils/conmon/patches/010-remove-libdl-dep.patch

```diff
@@ -1,26 +1,13 @@
---- a/meson.build
-+++ b/meson.build
-@@ -35,14 +35,6 @@ add_project_arguments('-Os', '-Wall', '-
- 
- glib = dependency('glib-2.0')
- 
--cc = meson.get_compiler('c')
--null_dep = dependency('', required : false)
--if cc.has_function('dlopen')
--  libdl = null_dep
--else
--  libdl = cc.find_library('dl')
--endif
--
- executable('conmon',
-            ['src/conmon.c',
-             'src/config.h',
-@@ -78,7 +70,7 @@ executable('conmon',
+Index: conmon-2.1.5/meson.build
```

Comment:
please refresh your patch with `make package/conmon/refresh V=s`

Reply:
Thanks for the advice. Done...